docs/user-manual/framework/memory-management/buffer-pool.md

```diff
@@ -263,20 +215,52 @@ Buffers bins should be tailored based on expected usage. If many small requests
 The above trivial example allows for a few small allocations and one large allocation. In this case, there is a risk that the large allocation is used for the small allocation use case and thus care should be taken to ensure that the smaller use
 cases have a sufficient number of buffers to prevent stealing of larger allocations.
 
-## Separation of `Fw::Buffer` Allocation and Deallocation
+## Deallocation strategy
 
-There is no requirement that the allocating component and the deallocating component are the same. Thus components may
-be chained together for multiple processing steps before deallocation. There are two requirements:
+`Fw::Buffer`s **must** eventually be returned to the same instance that allocated them.
 
-1. `Fw::Buffers` must eventually be returned to the instance that allocated them
-2. `Svc.StaticMemory` cannot be used in chains involving asynchronous calls
+Additionally, it is a recommended practice to deallocate a buffer in the same component where it was allocated. This is accomplished using the buffer ownership "Return-To-Sender" pattern described below.
 
-Inter-component connections typically use the same `Fw.BufferSend` port to pass the buffer along the chain. A sample
-chain is shown here using Svc.BufferManager as the allocation source.
+### Design Pattern: Return-To-Sender
```

Comment:
I feel like this should be pulled into a separate guide.  Return-To-Sender is needed in many scenarios involving ownership.....no just buffer pools.